### ISSUE TITLE
Exception when saving a new feed

### ISSUE THREAD

**Issue by tsegismont** [Sammers21/rss-reader#5]
```
GRAVE: Unhandled exception
java.lang.IllegalAccessError: tried to access method io.vertx.core.http.impl.HttpClientImpl.getVertx()Lio/vertx/core/impl/VertxInternal; from class io.vertx.ext.web.client.impl.HttpRequestImpl
        at io.vertx.ext.web.client.impl.HttpRequestImpl.send(HttpRequestImpl.java:230)
        at io.vertx.ext.web.client.impl.HttpRequestImpl.send(HttpRequestImpl.java:201)
        at rss.reader.FetchVerticle.lambda$start$3(FetchVerticle.java:49)
        at io.vertx.core.eventbus.impl.HandlerRegistration.deliver(HandlerRegistration.java:223)
        at io.vertx.core.eventbus.impl.HandlerRegistration.handle(HandlerRegistration.java:200)
        at io.vertx.core.eventbus.impl.EventBusImpl.lambda$deliverToHandler$3(EventBusImpl.java:533)
        at io.vertx.core.impl.ContextImpl.lambda$wrapTask$2(ContextImpl.java:339)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:463)
        at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:886)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by tsegismont** [comment]
I tried with name `Thomas` and feed `https://vertx.io/feed.xml`

**Comment by sandipan11** [comment]
What is the solution? you not clearly mentioned where you set name Thomas and feed https://vertx.io/feed.xml. Please elaborate.

**Comment by tsegismont** [comment]
@sandipan11 my problem was fixed in  af8786c

If you still hit this problem, please file another issue.
